common/src/main/java/com/lightstep/tracer/shared/Options.java

```diff
@@ -175,12 +169,6 @@ public Options withDisableReportingLoop(boolean disable) {
         return this;
     }
 
-    @SuppressWarnings("unused")
-    public Options withDisableReportOnExit(boolean disable) {
```

Comment:
This option is documented in `README.md` so that doc should be updated.  We should also likely start a CHANGELOG.md (or whatever the Java standard is) in the repo to document all changes that may require corresponding changes in the calling code.